# RegisterBlock.srv
sensor_msgs/Image       mask
sensor_msgs/PointCloud2 cloud
string                  object_class
---
geometry_msgs/Pose      pose
float64                 fitness
float64                 rmse
bool                    success
sensor_msgs/PointCloud2 cutout_cloud
sensor_msgs/PointCloud2 template_cloud
